en/install.rst

```diff
@@ -15,7 +15,7 @@ The platforms supported by CUBRID and hardware/software requirements for the ins
 +---------------------------------------------------------------------+------------------+---------------------+--------------------------------------+
 | Supported Platforms                                                 | Required Memory  | Required Disk Space | Required Software                    |
 +=====================================================================+==================+=====================+======================================+
-| * Windows 32/64 Bit Windows 7                                       | 1GB or more      | 2GB or more(\*)     | JRE/JDK 1.6 or higher                |
```

Comment:
Because this line was occurred to conflict with develop version, you should fix it.
After modifying, you will commit again.